CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [UNRELEASED] - YYYY-MM-DD
 
 ### Changed
-
+- [#25](https://github.com/equinor/flow-diagram-explorer/pull/25) - Make class names unique
```

Comment:
It would be nicer to have this in past tense, e.g.
`Changed class names to be unique for FlowDiagramExplorer.`